packages/start-plugin-core/src/rsbuild/import-protection.ts

```
@@ -0,0 +1,1589 @@
+import { resolve as resolvePath } from 'node:path'
```

ESLint

[error] 'node:path' imported multiple times.

(import/no-duplicates)